bdio2/src/main/resources/com/blackducksoftware/bdio2/bdio-context-2.1.jsonld

```diff
@@ -174,7 +174,6 @@
       "@id" : "https://blackducksoftware.github.io/bdio#hasShallowDirectoryCount",
       "@type" : "xsd:long"
     },
-    "scanType" : "https://blackducksoftware.github.io/bdio#hasScanType",
```

Comment:
Removed?

Reply:
` @type` is used to denote the scanType now, previously it was a data property. So it was removed when I generated the context using bdio-tool.